tests/integration/golang/migrations/migrations_test.go

```diff
@@ -0,0 +1,207 @@
+package migrations
+
+import (
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+	"gorm.io/driver/postgres"
+	"gorm.io/driver/sqlite"
+	"gorm.io/gorm"
+
+	"github.com/G-Research/fasttrackml/pkg/database"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/helpers"
+)
+
+type MigrationsTestSuite struct {
+	suite.Suite
+}
+
+func TestMigrationsTestSuite(t *testing.T) {
+	t.Skip()
```

Comment:
perhaps use build tag so we can run with argument but otherwise won't?

Reply:
I think this requires changing the source code to run it, yes? and then also it will run with all the others in that case, as w/ build tag. But with build tag, you can activate with just `go test` arguments, for CI.